Bio/PDB/mmtf/mmtfio.py

```diff
@@ -174,9 +191,11 @@ def _save_structure(self, filepath, select):
                         )
                         encoder.set_chain_info(
                             chain_id=next(chain_id_iterator),
-                            chain_name="\x00"
-                            if len(chain.get_id().strip()) == 0
-                            else chain.get_id(),
+                            chain_name=(
+                                "\x00"
+                                if len(chain.get_id().strip()) == 0
```

Comment:
I know this is not part of the PR but since you're here... this could be simplified to `"\x00" if not chain.id.strip() else chain.id.strip()`

Reply:
Ok